File: task-03/C++/code.cpp
 #include <iostream>

bool checkprime(int N) {
    if (N < 2) {
        return false;
    }

    for (int i = 2; i * i <= N; i++) {
        if (N % i == 0) {
            return false;
        }
    }
    
    return true;
}

using namespace std;

int main() {
    int n;
    cout << "Enter a number: ";
    cin >> n;

    for (int i = 2; i <= n; i++) {
        if (checkprime(i)) {
            cout << i << " ";
        }
    }
    
    cout << "\n" ;
    return 0;
}

File: task-03/Javascript/code.js
function checkPrime(n) {
    if (n < 2) {
        return false;
    }
    for (let i = 2; i < n; i++) {
        if (n % i === 0) {
            return false;
        }
    }
    return true;
}

const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question('Enter a number: ', input => {
    const N = parseInt(input);

    console.log(`Prime numbers up to ${N} are:`);

    for (let i = 2; i <= N; i++) {
        if (checkPrime(i)) {
            process.stdout.write(i + ' ');
        }
    }

    readline.close();
});
